dbt_subprojects/daily_spellbook/models/addresses/chains/addresses_ethereum_info.sql

```diff
@@ -0,0 +1,24 @@
+{% set blockchain = 'ethereum' %}
+
+{{
+    config(
+        schema = 'addresses_' + blockchain,
+        alias = 'info',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
```

Comment:
Maybe we can try `delete+insert` here.

```suggestion
        incremental_strategy = 'delete+insert',
```